cmd/ftl/cmd_serve.go

```diff
@@ -44,6 +44,7 @@ type serveCmd struct {
 	Stop                bool                 `help:"Stop the running FTL instance. Can be used with --background to restart the server" default:"false"`
 	StartupTimeout      time.Duration        `help:"Timeout for the server to start up." default:"1m"`
 	ObservabilityConfig observability.Config `embed:"" prefix:"o11y-"`
+	DatabaseImage string `help:"The container image to start for the database" default:"postgres:15.4" env:"FTL_DATABASE_IMAGE"`
```

Comment:
nit: formatting

Reply:
This still needs some work, it seems like something is still starting the old container

Reply:
Would `postgres:latest` be a better default? (not sure if that even works)

Reply:
The linked ticket was about aligning versions with the production database. One issue with using latest in general is that versions can change underneath you, meaning that code that previously worked can suddenly start to fail.